gralloc/gralloc.cpp

```diff
@@ -64,12 +77,38 @@ static int gbm_init(struct gbm_module_t *dmod)
 	return err;
 }
 
+static int get_fourcc(int format)
```

Comment:
Should this account for `persist.waydroid.invert_colors` like `get_gbm_format`?
Also gbm fourcc codes seem to be identical to drm, so we could directly use `get_gbm_format`

Reply:
Isn't the point of `persist.waydroid.invert_colors` to lie about used format?

Reply:
i don't know :shrug: